magento-compatibility.js

```diff
@@ -4,6 +4,7 @@
 
 // PWA Studio version -> Magento version.
 module.exports = {
+    '8.0.0': '2.3.6 - 2.4.1',
```

Comment:
I can see arguments for both sides of this but seeing as how our official support line has been "last two Magento releases", I think we should stick with that here.

It seems like a happy accident that anything more than that is compatible, and in this case specifically the reset password flow doesn't work. I do like that we've listed that as a known issue if we keep the current compatibility value.

Reply:
@supernova-at Agree completely. Support for `2.3.6` is coincidental, so I wouldn't list it.